surveyor.py

```diff
@@ -62,13 +71,16 @@ def cli(ctx, prefix, hostname, profile, days, minutes, product, username, iocfil
 
     # create a single connection to the appropriate product
     # for use in our calls
-    cb_conn = utils.get_cbapi_connection()
+    if product == "defender":
+        conn = utils.get_connection_creds(creds)
+    else: 
+        conn = utils.get_connection()
 
     # set the output file
     if output:
         output_file = open(output, 'w', newline='')
     elif prefix:
-        output_file = open(f'{prefix}-survey.csv', 'w', newline='')
+       output_file = open(f'{prefix}-survey.csv', 'w', newline='')
```

Comment:
Just want to make sure this indentation change is ok, as it doesn't match the `if` above it. Python can do some weird things when indentation isn't exact.